nomination/templates/nomination/project_urls.html

```diff
@@ -1,90 +1,73 @@
-{% extends "nomination/index.html" %}
-{% load staticfiles %}
+{% extends "nomination/base.html" %}
 
 {% block title %}Nomination Tool: Project URL Nomination{% endblock %}
 
-{% block extra_header %}
-{{ block.super }}
-<script type="text/javascript" src="{% static 'nomination/scripts/jquery.treeview.pack.js' %}"></script>
-<script type="text/javascript" src="{% static 'nomination/scripts/jquery.treeview.async.js' %}"></script>
-<!--
-<script type="text/javascript">
-    $(document).ready(function() {
-            $("#browse-list").treeview({
-                url: "{# url 'browse_json' project.project_slug,"surt" #}",
-                collapsed: true
-            });
-            $("#search-url-value").focus();
-
-    });
-</script>
--->
-{% endblock %}
-
-{% block main_nav %}{% endblock %}
-
 {% block content %}
-    <div id="gen-nom-stats">
-        <ul>
-            <li>
-                <a href="{% url 'url_report' project.project_slug  %}">
-                    Number of URLs Nominated:
+
+    <div class="panel-group">
+        {% include "nomination/proj_navbar.html" with active_tab='proj_urls' %}
+        <div class="panel panel-primary">
+            <div class="panel-heading">
+                <h4 class="panel-title">Search</h4>
+            </div>
+            <div class="panel-body">
+                <form method="post" role="form" action="{% url 'url_lookup' project.project_slug %}">{% csrf_token %}
+                    <label for="search-url-value">Search by URL</label>
+                    <div class="input-group col-md-6">
+                        <input type="text" class="form-control" name="search-url-value" id="search-url-value" aria-describedby="help-block" value="http://">
+                        <span class="input-group-btn">
+                            <button class="btn btn-primary" type="submit">submit</button>
+                        </span>
+                    </div>
+                    <span class="help-block" id="helpBlock">Search for an existing URL in the system.</span>
+                    <span>
+                        <input type="checkbox" name="partial-search" value="true">
+                        Allow partial matches
+                    </span>
+                </form>
+            </div>
+        </div>
+        <div class="panel panel-primary">
+            <div class="panel-heading">
+                <h4 class="panel-title">Browse URLs</h4>
+            </div>
+            <div class="panel-body">
+                <a href="{% url 'url_report' project.project_slug  %}" class="btn btn-info">
+                    <b>Number of URLs Nominated: </b>
```

Comment:
I would create a class for these labels and use a `<span>` to apply the class for `font-weight: bold` instead of using `<b>`. It's more semantically correct.